src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -15,7 +15,7 @@
     private Path anakinFilePath = Paths.get("data", "anakin.xml");
 
     public UserPrefs() {
-        setGuiSettings(500, 500, 0, 0);
+        setGuiSettings(690, 500, 0, 0);
```

Comment:
Just curious, what does this change do to the UI?

Reply:
This presents first-time users of our application with the most optimal screen dimensions for our application!